modules/http/http.c

```diff
@@ -708,7 +708,7 @@ http_dd_init(LogPipe *s)
                                        SYSLOG_NG_VERSION, curl_info->version);
 
 
-  return log_threaded_dest_driver_init_method(s);
+  return log_threaded_dest_driver_start_workers(&self->super);
```

Comment:
I like that the new convention has a similarity with LogWriter's update watches.